frontend/src/metabase/dashboard/components/DashCard/DashCardParameterMapper/DashCardCardParameterMapper.unit.spec.jsx

```diff
@@ -63,77 +63,105 @@ describe("DashCardParameterMapper", () => {
     ).toBeInTheDocument();
   });
 
-  it("should render an informative error state for action cards", () => {
-    setup({
-      dashcard: createMockActionDashboardCard(),
+  describe("Action cards", () => {
+    it("should render an informative error state for action cards", () => {
+      setup({
+        dashcard: createMockActionDashboardCard(),
+      });
+      expect(getIcon("info")).toBeInTheDocument();
+      expect(
+        screen.getByLabelText(/action settings to connect variables/i),
+      ).toBeInTheDocument();
     });
-    expect(getIcon("info")).toBeInTheDocument();
-    expect(
-      screen.getByLabelText(/action settings to connect variables/i),
-    ).toBeInTheDocument();
   });
 
-  it("should render an informative error state for link cards", () => {
-    const linkCard = createMockCard({ dataset_query: {}, display: "link" });
-    setup({
-      card: linkCard,
-      dashcard: createMockDashboardCard({
-        visualization_settings: {
-          virtual_card: linkCard,
-        },
-      }),
+  describe("Virtual cards", () => {
+    it("should render an informative error state for link cards", () => {
+      const linkCard = createMockCard({ dataset_query: {}, display: "link" });
+      setup({
+        card: linkCard,
+        dashcard: createMockDashboardCard({
+          visualization_settings: {
+            virtual_card: linkCard,
+          },
+        }),
+      });
+      expect(getIcon("info")).toBeInTheDocument();
+      expect(
+        screen.getByLabelText(/cannot connect variables to link cards/i),
+      ).toBeInTheDocument();
     });
-    expect(getIcon("info")).toBeInTheDocument();
-    expect(
-      screen.getByLabelText(/cannot connect variables to link cards/i),
-    ).toBeInTheDocument();
-  });
 
-  it("should render an informative parameter mapping state for text cards without variables", () => {
-    const textCard = createMockTextDashboardCard({ size_x: 3, size_y: 3 });
-    setup({
-      dashcard: textCard,
+    it("should render an informative parameter mapping state for text cards without variables", () => {
+      const textCard = createMockTextDashboardCard({ size_x: 3, size_y: 3 });
```

Comment:
Curios if `size_x` and `size_y` affect the tested behavior?

Reply:
only `size_y` matters

Reply:
removed size_x [f1a9811](https://github.com/metabase/metabase/pull/38545/commits/f1a981120db6760400349b5138332785843a1568)